blackbelt/apis/trello.py

```diff
@@ -130,15 +130,34 @@ def add_card_member(self, card_id, member_id):
             }   
         )
 
-    def label_card(self, card_id, label):
+    def label_card(self, card_id, label_id):
```

Comment:
I think you just broke labels for everyone else if they want to use colors instead of ids